src/utils/sanitizeText.js

```diff
@@ -0,0 +1,6 @@
+export function sanitizeText(text) {
+  return text
+    .replace(/<\/?[^>]+(>|$)/g, "") // Remove any HTML tags
+    .replace(/[^\w.?! ]/g, "") // Replace special characters or patterns
```

Comment:
does `Complete Registration Form #1` seem like a valid task name?

Reply:
No @sam-wtag vaia, as of now, the sanitization seem to remove the `#` symbol.